application/templates/home.html

```diff
@@ -0,0 +1,25 @@
+<html>
+	<head>
+		<title>Welcome to SantaBot</title>
+	</head>
+	
+	<body>
+		<h1>Greetings, Meatbag. Welcome to SantaBot.</h1>
+		<table>

```

Comment:
sometimes tables have a additional tags thead and tbody: https://www.w3schools.com/tags/tag_thead.asp. I think they can be useful in distinguishing a tr from a header vs body row.

Reply:
ok